src/test/resources/local-pipeline/run_local_pipeline

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+if [ $# -ne 5 ];
+then
+  echo "Usage: usgs_username usgs_password [landsat_5 | landsat_7 | landsat_8] region_pathrow image_date"
+  echo "For example: ./run_local_pipeline <usgs_username> <usgs_password> landsat_5 215065 2011-10-18"
+  exit 1
+fi
+
+USGS_USERNAME=$1
+USGS_PASSWORD=$2
+IMAGE_DATASET=$3
+IMAGE_REGION=$4
+IMAGE_DATE=$5
+
+IMAGE_NAME_INPUTDOWNLOADER="fogbow/inputdownloader:latest"
+IMAGE_NAME_WORKER="fogbow/worker:latest"
+
+CONTAINER_NAME_INPUTDOWNLOADER="inputdownloader_container"
+CONTAINER_NAME_WORKER="worker_container"
+
+function exit_handler {
+  if [[ $? -ne 0 ]];
+  then
+    exit 1
+  fi
+}
+
+function check_docker {
+  which docker
+  exit_handler
+}
+
+function pull_image_inputdownloader {
+  docker pull $IMAGE_NAME_INPUTDOWNLOADER
+}
+
+function pull_image_worker {
+  docker pull $IMAGE_NAME_WORKER
+}
+
+function build_container_inputdownloader {
+  docker ps -a | grep "$CONTAINER_NAME_INPUTDOWNLOADER"
+  if [ $? -eq 0 ];
+  then
+    docker stop $CONTAINER_NAME_INPUTDOWNLOADER
+    docker rm $CONTAINER_NAME_INPUTDOWNLOADER
+  fi
+
```

Comment:
add a comment explaining this. it is not crystal clear, so add a comment (remember your own struggle to understand some obscure SAPS/SEBAL code some days ago due to the missing comments/doc)